### ISSUE TITLE
eval semantickitt & test 

### ISSUE THREAD

**Issue by lzbushicai** [tsunghan-wu/RandLA-Net-pytorch#33]
when I run  evaluate_SemanticKITTI.py,the output is abnormal

When running test.py, there seems to be no output
`root@in_dev_docker RandLA-Net-pytorch $ python3 evaluate_SemanticKITTI.py --dataset lidar_seg/RandLA-Net-pytorch/data/nvme1/SemanticKITTI/randla_data --predictions lidar_seg/RandLA-Net-pytorch/runs/kitti --sequence 8
********************************************************************************
INTERFACE:
Eval What: None
Data:  /home/liuzhen/lidar_seg/RandLA-Net-pytorch/data/nvme1/SemanticKITTI/randla_data
Predictions:  /home/liuzhen/lidar_seg/RandLA-Net-pytorch/runs/kitti
Sequences:  ['8']
Config:  utils/semantic-kitti.yaml
Limit:  None
********************************************************************************
Opening data config file utils/semantic-kitti.yaml
Ignoring xentropy class  0  in IoU evaluation
[IOU EVAL] IGNORE:  [0]
[IOU EVAL] INCLUDE:  [ 1  2  3  4  5  6  7  8  9 10 11 12 13 14 15 16 17 18 19]
Evaluating sequences
0it [00:00, ?it/s]
Validation set:
Acc avg 0.000
IoU avg 0.000
IoU class 1 [car] = 0.000
IoU class 2 [bicycle] = 0.000
IoU class 3 [motorcycle] = 0.000
IoU class 4 [truck] = 0.000
IoU class 5 [other-vehicle] = 0.000
IoU class 6 [person] = 0.000
IoU class 7 [bicyclist] = 0.000
IoU class 8 [motorcyclist] = 0.000
IoU class 9 [road] = 0.000
IoU class 10 [parking] = 0.000
IoU class 11 [sidewalk] = 0.000
IoU class 12 [other-ground] = 0.000
IoU class 13 [building] = 0.000
IoU class 14 [fence] = 0.000
IoU class 15 [vegetation] = 0.000
IoU class 16 [trunk] = 0.000
IoU class 17 [terrain] = 0.000
IoU class 18 [pole] = 0.000
IoU class 19 [traffic-sign] = 0.000
********************************************************************************
0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000`
